spec/darlingtonia/parser_spec.rb

```diff
@@ -24,12 +24,14 @@ def match?(**_opts)
             end
           end
         end
+
+        class NestedParser < FakeParser; end
       end
 
       after(:context) { Object.send(:remove_const, :FakeParser) }
```

Comment:
Oh, make me fix this, I need to undef `NestedParser` too.

Reply:
heh, okay, fix this too :-)